tensorrt_llm/_torch/models/checkpoints/hf/checkpoint_loader.py

```diff
@@ -0,0 +1,77 @@
+from typing import Optional
+
+from tensorrt_llm._torch.models.checkpoints.base_checkpoint_loader import \
+    BaseCheckpointLoader
+from tensorrt_llm._torch.models.checkpoints.base_config_loader import \
+    BaseConfigLoader
+from tensorrt_llm._torch.models.checkpoints.base_weight_loader import \
+    BaseWeightLoader
+from tensorrt_llm._torch.models.checkpoints.base_weight_mapper import \
+    BaseWeightMapper
+from tensorrt_llm._torch.models.checkpoints.hf.config_loader import \
+    HfConfigLoader
+from tensorrt_llm._torch.models.checkpoints.hf.weight_loader import \
+    HfWeightLoader
+from tensorrt_llm._torch.models.modeling_utils import register_checkpoint_loader
+
+
+@register_checkpoint_loader("HF")
+class HfCheckpointLoader(BaseCheckpointLoader):
+
+    def __init__(self,
```

Comment:
Force passing by name (change this in baseclass as well):
```suggestion
    def __init__(self, *,
```